### ISSUE TITLE
A question about task prompt

### ISSUE THREAD

**Issue by wangbo-zhao** [baaivision/Painter#2]
A perfect paper. But I am a little confused about this:
“The second baseline is to generate a task prompt. We define the task prompt as the learnable tensors, freeze the whole model, and then use the training loss to optimize the task prompts.”
Could you give us more details about training the task prompt?

**Comment by WXinlong** [comment]
Hi Wangbo, the training of prompt tuning is the same as the end-to-end setting except 1) freeze all parameters but the prompt image, 2) use a larger learning rate, e.g., 1e-1, and 3) use a shorter training schedule.

**Comment by MinglangQiao** [comment]
@WXinlong , Hi, thanks for your great work. I have an additional question: does the freezed model for prompt tuning mean the untrained vit_large' model or the trained painter model?
